docs/sources/send-data/promtail/installation.md

```diff
@@ -17,6 +17,10 @@ or there is a Helm chart to install it in a Kubernetes cluster.
 Every Grafana Loki release includes binaries for Promtail which can be found on the
 [Releases page](https://github.com/grafana/loki/releases) as part of the release assets. 
 
+## Install using APT or RPM package manager
+
+See [here](../../setup/install/local.md). 
```

Comment:
```suggestion
See the instructions [here](https://grafana.com/docs/loki//setup/install/local/#install-using-apt-or-rpm-package-manager). 
```

Reply:
Is the `/latest/` inserted here automatically?